.github/workflows/codestyle.yaml

```diff
@@ -71,7 +71,7 @@ jobs:
       run: |
         set -eE
         echo "Commit ${{ github.event.pull_request.base.sha }}"
-        diff=`git clang-format --binary=clang-format-21 --style=file --diff ${{ github.event.pull_request.base.sha }}`
+        diff=`git-clang-format-21 --binary=clang-format-21 --style=file --diff ${{ github.event.pull_request.base.sha }}`
```

Comment:
not sure this is correct fix, git clang format will use proper script but probably clang format is installed as clang-format-21 which causes issues

Reply:
enhanced installation process